sections/semantics-interactive-elements.include

```diff
@@ -152,19 +167,28 @@
     Because the <{details/open}> attribute is added and removed
     automatically as the user interacts with the control, it can be used in CSS to style the element
     differently based on its state. Here, a stylesheet is used to animate the color of the summary
-    when the element is opened or closed:
-
-    <pre highlight="html">
-      &lt;style&gt;
-        details &gt; summary { transition: color 1s; color: black; }
-        details[open] &gt; summary { color: red; }
-      &lt;/style&gt;
-      &lt;details&gt;
-        &lt;summary&gt;Automated Status: Operational&lt;/summary&gt;
-        &lt;p&gt;Velocity: 12m/s&lt;/p&gt;
-        &lt;p&gt;Direction: North&lt;/p&gt;
-      &lt;/details&gt;
-    </pre>
+    when the element is opened or closed.
+
+    CSS:
+    <xmp highlight="css">
+      details>summary {
```

Comment:
May the selector spacing be preserved here? e.g. `details > summary`. That would be more readable and consistent with the choice to preserve spacing between properties and values.

Reply:
Agreed. Note to self - https://www.w3.org/TR/CSS2/selector.html#child-selectors